core/src/main/java/com/netease/arctic/io/reader/GenericCombinedIcebergDataReader.java

```diff
@@ -305,5 +346,14 @@ protected InputFile getInputFile(String location) {
     protected ArcticFileIO getArcticFileIo() {
       return fileIO;
     }
+
+    @Override
+    protected CloseableIterable<Record> readIdentifierData(Schema deleteSchema) {
+      return CloseableIterable.concat(
+          CloseableIterable.transform(
+              CloseableIterable.withNoopClose(
+                  Arrays.stream(input.rewrittenDataFiles()).collect(Collectors.toList())),
```

Comment:
Not only rewrittenDataFiles but also rePosDeletedDataFiles

Reply:
In the rePosDeletedDataFiles scenario, the Bitmap used has been optimized, and there should be no performance bottleneck for the time being. The current mode will only start with rewrittenDataFiles. 
WDYT?

Reply:
rePosDeletedDataFiles is data file not position-delete